src/main/java/no/sikt/nva/scrapers/DublinCoreScraper.java

```diff
@@ -445,7 +328,7 @@ private static String handleIsbnList(List<String> isbnList, BrageLocation brageL
             logger.warn("Following resource contains many isbn values: " + brageLocation.getOriginInformation());
         }
         if (isbnList.isEmpty()) {
-            return new DcValue().getValue();
+            return StringUtils.EMPTY_STRING;
         }
         return isbnList.get(0);
     }
```

Comment:
Er du sikker på at vi ønsker å bruke emtpy string her? for da blir det opprettet en issn og isbn på en record som en tom streng.

  "publication" : {
    "issn" : "",
    "isbn" : "9788241010347",
    "publisher" : "NVE"
  }